### ISSUE TITLE
New Accessibility (a11y) emoji key

### ISSUE THREAD

**Issue by phacks** [all-contributors/all-contributors#320]
**Is your feature request related to a problem? Please describe.**

On https://github.com/theodo/falco, we are seeing a few contributors reporting a11y issues and/or pushing a11y-related pull requests. I feel like those contributions should be properly rewarded by a specific emoji key.

**Describe the solution you'd like**

A new emoji key, called `a11y` or `accessibility`:

Emoji/Type | Represents | Comments
:---: | :---: | :---:
♿️ <br /> `a11y` | Accessibility | Reporting or working on accessibility issues

**Describe alternatives you've considered**

I’m not convinced by my choice of emoji, here are a few other suggestions: 🕊🙌✨

**Comment by Berkmann18** [comment]
That's a great idea.
The :wheelchair: seems to be the best for this as far as I can tell.

@all-contributors/core Any thoughts/objections?

**Comment by gr2m** [comment]
I love the idea.

**Comment by phacks** [comment]
Cool! Thanks for the feedbacks. Should I go forward and open a PR?

**Comment by gr2m** [comment]
Yes, thank you

**Comment by phacks** [comment]
PRs are ready!

- https://github.com/all-contributors/all-contributors-cli/pull/223
- https://github.com/all-contributors/all-contributors/pull/326

Please let me know if I forgot something :)

**Comment by adswa** [comment]
Hey everyone, thanks for adding this wonderful contribution type! I'm really happy to see this!

I've been trying to use this, but for some reason, I fail ([this conversation](https://github.com/datalad-handbook/book/pull/253)):
![image](https://user-images.githubusercontent.com/29738718/69701296-e9428b00-10ec-11ea-832f-f88bbff2aaaa.png)

What am I doing wrong? Thanks a lot in advance!

**Comment by sinchang** [comment]
@adswa I guess that feature still not released.

**Comment by adswa** [comment]
Gotcha - thanks @sinchang!

**Comment by Berkmann18** [comment]
It was merged in the bot repo (cf. https://github.com/all-contributors/all-contributors-bot/pull/259) so normally it should be available.

**Comment by jakebolam** [comment]
![Screen Shot 2019-11-27 at 1 03 12 PM](https://user-images.githubusercontent.com/3534236/69748416-4a328900-1116-11ea-92fb-b898de692a1f.png)

Ahh sorry, the pipeline was [stopped on verify](https://circleci.com/workflow-run/8c5d5684-53d1-4a95-bb2e-27fe6a04de7c).

It's on the sandbox https://github.com/all-contributors-sandbox

**Comment by jakebolam** [comment]
Ah the sandbox actually has an issue:
https://sentry.io/organizations/all-contributors/issues/957977140/?project=1366866&query=is%3Aunresolved

**Comment by jakebolam** [comment]
Fix is up: https://github.com/all-contributors/all-contributors-bot/pull/263

**Comment by jakebolam** [comment]
Fix has been released.

**Comment by KirstieJane** [comment]
I think this issue can close - it looks like everything's fixed?

**Comment by Berkmann18** [comment]
@KirstieJane Yup, thanks for the reminder.
